src/SAML2/XML/md/EntityDescriptor.php

```diff
@@ -450,16 +451,16 @@ public function addAdditionalMetadataLocation(AdditionalMetadataLocation $additi
      */
     public function toXML(\DOMElement $parent = null)
     {
-        assert(is_string($this->getEntityID()));
-        assert(is_null($this->getID()) || is_string($this->getID()));
-        assert(is_null($this->getValidUntil()) || is_int($this->getValidUntil()));
-        assert(is_null($this->getCacheDuration()) || is_string($this->getCacheDuration()));
-        assert(is_array($this->getExtensions()));
-        assert(is_array($this->getRoleDescriptor()));
-        assert(is_null($this->getAffiliationDescriptor()) || $this->getAffiliationDescriptor() instanceof AffiliationDescriptor);
-        assert(is_null($this->getOrganization()) || $this->getOrganization() instanceof Organization);
-        assert(is_array($this->getContactPerson()));
-        assert(is_array($this->getAdditionalMetadataLocation()));
+        Assert::string($this->getEntityID());
+        Assert::nullOrString($this->getID());
+        Assert::nullOrInteger($this->getValidUntil());
+        Assert::nullOrString($this->getCacheDuration());
+        Assert::isArray($this->getExtensions());
+        Assert::isArray($this->getRoleDescriptor());
+        Assert::nullOrIsInstanceOf($this->getAffiliationDescriptor());
+        Assert::nullOrIsInstance($this->getOrganization());
```

Comment:
Does `nullOrIsInstance()` exist, or is this a typo?

Reply:
That's a typo, thanks!